src/frameworks/drupal8/elasticsearch.md

```diff
@@ -57,9 +57,12 @@ Then, paste the following code snippet into your `settings.platformsh.php` file.
 ```php
 // Update these values to the relationship name (from .platform.app.yaml)
 // and the machine name of the server from your Drupal configuration.
-$relationship_name = 'elasticsearch';
+$relationship_name = 'RELATIONSHIP_NAME';
```

Comment:
I'd prefer to leave this at the value that is most commonly used, so it doesn't need to be modified.

Reply:
Fair enough, however it doesn't match https://docs.platform.sh/configuration/services/elasticsearch.html#usage-example.